context/app/markdown/docs.md

```diff
@@ -9,14 +10,17 @@
 - [2020 Summer Release](/docs/release-summer-2020)
 - [Frequently Asked Questions](/docs/faq)
 
----
+
+## Data 
 - [Data States](/docs/data-states)
 - [Metadata](/docs/metadata)
 - [Assays](/docs/assays)
 - [Pipelines](/docs/pipelines)
 - [CCF](https://hubmapconsortium.github.io/ccf/)
 
----
+
+## Technical Documentation 
+- [Software]
```

Comment:
```suggestion
- [Software](/docs/software)
```
Merging docs update into the branch: I'll run it locally to make sure the link works.